src/ape/api/accounts.py

```diff
@@ -98,6 +99,8 @@ def call(
             txn (:class:`~ape.api.transactions.TransactionAPI`): An invoke-transaction.
             send_everything (bool): ``True`` will send the difference from balance and fee.
               Defaults to ``False``.
+            private (bool): ``True`` will use the `eth_sendPrivateTransaction` RPC method.
+              Provider needs to implement `send_private_transaction` method.
             **signer_options: Additional kwargs given to the signer to modify the signing operation.
```

Comment:
- [ ] this method should include a `Raises` section in the docs (see other methods in codebase for example) to say it may raise `APINotImplementedError` from `ape.exceptions`